src/systems/filecoin_mining/storage_mining/storage_miner_actor_code.go

```diff
@@ -526,21 +530,22 @@ func (a *StorageMinerActorCode_I) _verifySeal(rt Runtime, onChainInfo sector.OnC
 	h, st := a.State(rt)
 	info := st.Info()
 	sectorSize := info.SectorSize()
-	dealIDs := onChainInfo.DealIDs()
-	params := make([]util.Serialization, 1+len(dealIDs))
+
+	// @param sectorSize util.UVarint
+	// @param dealIDs []deal.DealID onChainInfo.DealIDs()
+	getPieceInfoParams := make([]util.Serialization, 2)
 	panic("TODO: serialize arguments")
```

Comment:
maybe can take care of above using the auto-generated serialization method

Reply:
ack, will do